### PR TITLE
Fix TechDocs sidebar positions

### PR DESCRIPTION
**Pull request by roflc0pter**
## Hey, I just made a Pull Request!

As part of this PR https://github.com/backstage/backstage/pull/5640, it fixes the overlapping of the sidebars on the tab-bar.
The `querySelector` method is applied on the wrong element. It should be called inside the shadow-root tree. It is on the wrapper instead which didn't function.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by yousifalraheem — APPROVED** [review]

**Review by OrkoHunter — APPROVED** [review]
Any screenshots would definitely help reviewing changes like these. :)

File: .changeset/many-olives-eat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Fix on shadow-dom selector
```
Comment (Rugvip) [inline .changeset/many-olives-eat.md:5]: Perhaps this could describe a bit more around what was fixed rather than how it was fixed?

**Review by Rugvip — COMMENTED** [review]
Leaving review of the code to @backstage/techdocs-core, just a nit on the changeset 😅

**Comment by OrkoHunter** [comment]
Closing in favor of https://github.com/backstage/backstage/pull/6121

**Pull request closed without merging**